order/src/main/java/org/order/OrderController.java

```diff
@@ -0,0 +1,37 @@
+package org.order;
+
+import java.util.List;
+import lombok.AllArgsConstructor;
+import org.order.dto.PurchaseOrderDto;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@AllArgsConstructor
+public class OrderController {
+
+    private final OrderService orderService;
+
+    @ResponseStatus(HttpStatus.CREATED)
+    @PostMapping("/orders")
+    public Long addOrder(@RequestBody PurchaseOrderDto purchaseOrderDto) {
+        return orderService.addOrder(purchaseOrderDto);
+    }
+
+    @GetMapping("/orders")
+    List<PurchaseOrderDto> findAllOrders() {
+        return orderService.findAll();
+    }
+
+    @DeleteMapping("/orders/{id}")
```

Comment:
Consider that you are deleting only one single order, then it should be on singular
```suggestion
    @DeleteMapping("/order/{id}")
```

Reply:
as above